cmake/libssh2-config.cmake.in

```diff
@@ -1,6 +1,29 @@
 # Copyright (C) The libssh2 project and its contributors.
 # SPDX-License-Identifier: BSD-3-Clause
 
+include(CMakeFindDependencyMacro)
+list(PREPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+
+if("@CRYPTO_BACKEND@" STREQUAL "OpenSSL")
+  find_dependency(OpenSSL)
+endif()
+
+if("@CRYPTO_BACKEND@" STREQUAL "wolfSSL")
+  find_package(wolfssl)
+endif()
+
+if("@CRYPTO_BACKEND@" STREQUAL "Libgcrypt")
+  find_package(Libgcrypt)
+endif()
+
+if("@CRYPTO_BACKEND@" STREQUAL "mbedTLS")
+  find_package(mbedTLS)
+endif()
```

Comment:
Thanks! Looking good.
I'm wondering if using `elseif()` would make this look sweeter, what do you think?

Reply:
I also think `elsif()` would be nice. Wait a moment.